### DISCUSSION TITLE
Fully Online Student

### DISCUSSION THREAD

**Discussion by tramsayUT** [community/community#131265] [GitHub Education]
### Hi everyone,

I am enrolled in a fully online data science program through UT Austin and do not have any physical copies of proof of enrollment or other physical documents from UT. I have a PDF of my proof of enrollment and have tried to submit a picture of it from my phone, but it was rejected immediately. Is GitHub Education open to fully online students like me, and is there a way I can upload my PDF so it can be reviewed?

**Comment by davevad93** [comment]
Hi @tramsayUT ,

GitHub Education doesn't make individual manual reviews and only allows uploading a picture of the documentation required for the application, however it does allow applications from distance learners who are part of a verifiable distance learning program. If your school's has some missing or incorrect info you can open a ticket to [Education support ](https://support.github.com/contact/education) and select `My selected school has incorrect or incomplete information e.g. domains or campus location`.

For more info, take a look at the following post about [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352).
